miniapp/src/test/java/com/rakuten/tech/mobile/miniapp/display/RealMiniAppDisplayTest.kt

```diff
@@ -18,6 +19,7 @@ class RealMiniAppDisplayTest {
     private lateinit var context: Context
     private lateinit var basePath: String
     private lateinit var realDisplay: RealMiniAppDisplay
+    private val masInf: MiniAppMessageInterface = com.nhaarman.mockitokotlin2.mock()
```

Comment:
How about using import for `com.nhaarman.mockitokotlin2.mock()` and just using mock() like everywhere else I think?